### DISCUSSION TITLE
how do i install this my add is tarver#0001

### DISCUSSION THREAD

**Discussion by lolman213** [marioparaschiv/nitro-sniper#105] [Q&A] [answered]
pls can anyone help me install im on windows ty

**Comment by Jorgen10** [comment, marked as answer]
https://github.com/slow/nitro-sniper/wiki/Installation

Download Git and use the steps in here

**Reply by lolman213** [reply]
i have im just not smart nor understand do u mind sending a copy with all fills in there
